application.py

```diff
@@ -0,0 +1,31 @@
+"""Application factory and configuration for azure_lightning_flask applications"""
+
+from logging import getLogger, StreamHandler
+from flask import Flask
+from flask_env import MetaFlaskEnv
+from azure_lightning_flask.application_blueprint import application_blueprint
+
+class Configuration(object): # pylint: disable=too-few-public-methods
+    """Config merging environment variables into flask config using flask-env"""
+    __metaclass__ = MetaFlaskEnv
+
+    # Default values for configuration taken from ember-cli-deploy-azure
+    AZURE_STORAGE_TABLE = 'emberdeploy'
+    AZURE_STORAGE_TABLE_PARTITION_KEY = 'manifest'
+    REVISION_PARAMETER = 'index_key'
+
+def create_app(config=Configuration):
+    """Return azure_lightning_flask application using given configuration"""
+    app = Flask(__name__)
+
+    app.config.from_object(config)
+
+    # Setup logging for Azure network requests
+    azure_logger = getLogger('azure.storage.common.storageclient')
+    azure_logger_handler = StreamHandler()
+    azure_logger_handler.setLevel(app.logger.getEffectiveLevel())
+    azure_logger.addHandler(azure_logger_handler)
+
+    app.register_blueprint(application_blueprint)
+
+    return app
```

Comment:
I think it could be nice to output an error and close the flask app if the environment variables that need to be configured are empty. `Error: environment variable AZURE_STORAGE_ACCOUNT must be set!`

Reply:
RESOLVED